onnxruntime/core/providers/dml/OperatorAuthorHelper/OperatorHelper.cpp

```diff
@@ -56,6 +56,17 @@ namespace OperatorHelper
         }
     }
 
+    template <typename T>
+    void ExpandToAxes(/*inout*/ std::vector<T>& attr, std::vector<int32_t> axes, std::vector<T> expanded)
```

Comment:
Assertions will clarify expectations for later readers.

`assert(originalValues.size() == axes.size())` or `ML_INVALID_ARGUMENT`